samples: openthread: no need for separated nRF53 cert file (#3854)
nrfconnect/sdk-nrf: doc/nrf/ug_thread_certification.rst

#. Build the certification image.

>    The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.

Reviewer: You need a blank line after this and before the code-block, or it will not show up correctly.

#. Build the certification image.

>    The :ref:`ot_cli_sample` sample is used as a base, modified with an overlay file: :file:`harness/overlay-cert.conf`.

Reviewer: ```suggestion
   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
```

   .. note::
      The overlay file selects the precompiled OpenThread libraries.

>       This file also enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth LE advertising.

Reviewer: ```suggestion
      It also enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth LE advertising.
```
